package.json

```diff
@@ -145,7 +145,6 @@
     "mock-require": "^3.0.3",
     "nyc": "^14.1.1",
     "pkg": "^4.4.2",
-    "prettier": "^1.19.1",
```

Comment:
There should be no changes to `package.json`

Reply:
Thanks for review @medikoo,  just fixed the version of prettier.